File: projeto/unicafeweb/unicafeWeb/classes/modelo/Acesso.php
<?php
class Acesso{
	
	private $id;
	private $tempoUsado;
	private $horaInicial;
	private $usuario;
	private $tempoDisponibilizado;
	private $status;
	
	
	public function Acesso(){
		$this->usuario = new Usuario();
	}
	public function setId($id){
		$this->id = $id;
	}
	public function getId(){
		return $this->id;
	}
	public function setTempoUsado($tempoUsado){
		$this->tempoUsado = $tempoUsado;
	}
	public function getTempoUsado(){
		return $this->tempoUsado;
	}
	public function setHoraInicial($horaInicial){
		$this->horaInicial = $horaInicial;
	}
	public function getHoraInicial(){
		return $this->horaInicial;
	}
	public function setUsuario(Usuario $usuario){
		$this->usuario = $usuario;
	}
	public function getUsuario(){
		return $this->usuario;
	}
	public function setTempoDisponibilizado($tempoDisponibilizado){
		$this->tempoDisponibilizado = $tempoDisponibilizado;
	}
	public function getTempoDisponibilizado(){
		return $this->tempoDisponibilizado;
	}
	public function setStatus($status){
		$this->status = $status;
	}
	public function getStatus(){
		return $this->status;
	}
	
	
	
	
}

File: src/UniCaffe/unicafe/src/main/java/br/edu/unilab/unicafe/model/Acesso.java
package br.edu.unilab.unicafe.model;

public class Acesso {
	
	
	private int id;
	private int tempoUsado;
	private long horaInicial;
	private Usuario usuario;
	private int tempoDisponibilizado;
	private int status;
	private int idMaquina;
	private String ip;
	public static final int STATUS_EM_UTILIZACAO = 0;
	public static final int STATUS_DISPONIVEL = 1;

	
	public Acesso() {
		this.status = STATUS_DISPONIVEL;
		this.usuario = new Usuario();
		this.setTempoUsado(0);
	}
	public void contar(){
		this.status = STATUS_EM_UTILIZACAO;
		Thread contando = new Thread(new Runnable() {
			@Override
			public void run() {
				while(status == STATUS_EM_UTILIZACAO){
					try {
						Thread.sleep(1000);
						tempoUsado++;
					} catch (InterruptedException e) {
						
					}
				}
			}
		});
		contando.start();
	}
	public void pararDeContar(){
		this.status = STATUS_DISPONIVEL;
	}
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public Usuario getUsuario() {
		return usuario;
	}
	public void setUsuario(Usuario usuario) {
		this.usuario = usuario;
	}
	public int getTempoUsado() {
		return tempoUsado;
	}
	public void setTempoUsado(int tempoUsado) {
		this.tempoUsado = tempoUsado;
	}
	public long getHoraInicial() {
		return horaInicial;
	}
	public void setHoraInicial(long l) {
		this.horaInicial = l;
	}
	public int getTempoDisponibilizado() {
		return tempoDisponibilizado;
	}
	public void setTempoDisponibilizado(int tempoDisponibilizado) {
		this.tempoDisponibilizado = tempoDisponibilizado;
	}
	public int getStatus() {
		return status;
	}
	public void setStatus(int statusDaConexao) {
		this.status = statusDaConexao;
	}
	
	
	@Override
	public String toString() {
		String acesso = "usuario: "+this.usuario.getId()+", hora Acesso Long: "+this.horaInicial+", Tempo Usado: "+this.tempoUsado+", "+this.tempoDisponibilizado+"BLZ\n";
		return acesso;
	}
	public int getIdMaquina() {
		return idMaquina;
	}
	public void setIdMaquina(int idMaquina) {
		this.idMaquina = idMaquina;
	}
	public String getIp() {
		return ip;
	}
	public void setIp(String ip) {
		this.ip = ip;
	}

	
}
